network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/tck/MergeNetworkTest.java

```diff
@@ -39,32 +40,117 @@ public void multipleDanglingLinesInMergedNetwork() {
     }
 
     @Override
-    public void testMergeSameId() {
+    public void test() {
         // FIXME
     }
 
     @Override
-    public void test() {
+    public void mergeThenCloneVariantBug() {
         // FIXME
     }
 
     @Override
-    public void xnodeNonCompatible() {
+    public void multipleDanglingLinesInMergingNetwork() {
         // FIXME
     }
 
     @Override
-    public void mergeThenCloneVariantBug() {
+    public void checkMergingSameFormat() {
         // FIXME
     }
 
     @Override
-    public void multipleDanglingLinesInMergingNetwork() {
+    public void testMergeAndDetach() {
         // FIXME
     }
 
     @Override
-    public void checkMergingSameFormat() {
+    public void testMergeAndDetachWithExtensions() {
+        // FIXME
+    }
+
+    @Override
+    public void failDetachWithALineBetween2Subnetworks() {
+        //FIXME
+    }
+
+    @Override
+    public void failDetachIfMultiVariants() {
+        //FIXME
+    }
+
+    @Test
```

Comment:
It's a problem that those methods are protected in powsybl-core nope ?

Reply:
I don't see what the problem would be ?